main.go

```diff
@@ -61,9 +66,18 @@ func mainErr() error {
 		return errors.Wrap(err, "could not read web-identity-token from file")
 	}
 
+	var sessName string
+	if sessNameEnv := os.Getenv(AwsSessionName); sessNameEnv != "" {
+		sessName = sessNameEnv
+	} else if *sessionName == "" {
+		sessName = uuid.New().String()
+	} else {
+		sessName = *sessionName
+	}
```

Comment:
```suggestion
	} else if *sessionName.isNotEmpty() {
	         sessName = *sessionName	
	} else {
		sessName = uuid.New().String()
	}
```

is more readable an intuitive

Reply:
I don't know, if there is a "isNotEmpty" method ;)